app/src/androidTest/java/com/github/orkest/ShareSongTest.kt

```diff
@@ -0,0 +1,73 @@
+package com.github.orkest
+
+import com.github.orkest.Model.Authorization
+import org.junit.Test
+import android.content.Intent
+import androidx.lifecycle.Lifecycle
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.matcher.IntentMatchers.*
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import com.github.orkest.View.sharing.SharingComposeActivity
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+
+
+
+class SpotifyAuthorizationTest {
+
+    @get:Rule
+    val activityRule = ActivityScenarioRule(SharingComposeActivity::class.java)
+
+    @Before
+    fun setUp() {
+        // Initialize the Intents class
+        Intents.init()
+    }
+
+    @After
+    fun tearDown() {
+        // Release the resources used by the Intents class
+        Intents.release()
+    }
```

Comment:
That's a good practice. We should do that for other test classes that require to release the resources.